docs/src/domains/in-depth/_index.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: "Going Live - Steps"
+title: "Custom Domains - In-depth"
 weight: 2
-sidebarTitle: "Go-live steps"
+sidebarTitle: "Detailed Step"
 description: |
-  Going live on Platform.sh is a simple two or three step process. You can either use the Platform.sh management console or the CLI to configure your project for production. Once you have gone through it once the whole process usually takes a couple of minutes.
+  Configuring custom domains on Platform.sh is a simple two or three step process. You can either use the Platform.sh management console or the CLI to configure your project for production. Once you have gone through it once the whole process usually takes a couple of minutes.
```

Comment:
"Once you have gone through it once" is grammatically valid but feels clumsy.  Let's clean that up while we're here.

Reply:
```suggestion
  Configuring custom domains on Platform.sh is a simple two or three step process. You can either use the Platform.sh management console or the CLI to configure your project for production. Once you are familiar with it the whole process usually takes a couple of minutes.
```

Reply:
Thank you Crell.